integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/withapim/PrototypedAPITestCase.java

```diff
@@ -91,6 +91,6 @@ public void invokePrototypeAPISuccessTest() throws Exception {
                         Utils.getServiceURLHttps("/petstore-prototype/1.0.0/pet/findByStatus"), headers);
 
         Assert.assertNotNull(response);
-        Assert.assertEquals(response.getResponseCode(), HttpStatus.SC_OK,"Response code mismatched");
```

Comment:
btw, don't we now need a success test as well?

Reply:
added via https://github.com/wso2/product-microgateway/pull/3117/commits/5065f20a52d7146c6639637f0ecdc71023311b34